tests/test_client_functional.py

```diff
@@ -673,6 +673,47 @@ async def handler(request):
             await resp.content.read()
 
 
+async def test_read_timeout_on_sending_chunks(aiohttp_client: Any) -> None:
```

Comment:
This test will fail without the changes in `StreamWriter.write`, `StreamWriter.write_eof`, as reported in #7149